resources/views/contact.blade.php

```diff
@@ -29,80 +29,96 @@
                                     <div class="input-group">
-                                        <input type="text" class="form-control" placeholder="Phone No.">
+                                        <input type="text" class="form-control" placeholder="Phone No." name="phone" value="{{ old('phone') }}">
                                     </div>
                                 </div>
                                 <div class="col-sm-6">
                                     <div class="input-group">
-                                        <input type="text" class="form-control" placeholder="Project Title">
+                                        <input type="text" class="form-control" placeholder="Project Title" name="project" value="{{ old('project') }}">
                                     </div>
                                 </div>
+                                
                                 <div class="col-sm-6">
                                     <div class="input-group">
-                                        <select class="form-control">
+                                        <select class="form-control" name="service">
                                             <option selected>Choose Service</option>
-                                            <option value="1">Web Development</option>
-                                            <option value="2">Web Design</option>
-                                            <option value="3">Strategy & Research</option>
+                                            <option value="Web Development" {{old('service') == 'Web Development' ? 'selected' : '' }}  >Web Development</option>
+                                            <option value="Web Design" {{old('service') == 'Web Design' ? 'selected' : '' }}>Web Design</option>
+                                            <option value="Strategy & Research" {{old('service') == 'Strategy & Research' ? 'selected' : '' }}>Strategy & Research</option>
                                         </select>
                                     </div>
                                 </div>
                                 <div class="col-sm-12">
                                     <div class="input-group">
-                                        <textarea class="form-control">Message</textarea>
+                                        <textarea name="message"  class="form-control" placeholder="Your Message...">{{ old('message') }}</textarea>
                                     </div>
                                 </div>
-                                <div class="col-sm-12">
+                                <!-- <div class="col-sm-12">
                                     <div class="input-group custom-file">
                                         <input type="file" class="form-control custom-file-input">
                                         <label class="custom-file-label">Choose file</label>
```

Comment:
do we need a file uploader? if so , better to set a filtering to allow only specific types to limit security vulnerability

Reply:
We are not planning for file uploads now.